src/resolvers/registries/npm-resolver.js

```diff
@@ -24,6 +24,10 @@ export default class NpmResolver extends RegistryResolver {
   static registry = 'npm';
 
   static async findVersionInRegistryResponse(config: Config, range: string, body: RegistryResponse): Promise<Manifest> {
+    if (!body['dist-tags']) {
```

Comment:
I was thinking the same thing, but I believe this is possibly the best indicator of a well-formed response, given what we want. Definitely open to suggestions, but I was specifically trying to avoid an error slightly lower down where `body['dist-tags']` is `undefined`.